featuretools/tests/synthesis/test_dfs_method.py

```diff
@@ -212,20 +212,13 @@ def test_accepts_pd_timedelta_training_window(datetime_es):
 def test_accepts_pd_dateoffset_training_window(datetime_es):
     feature_matrix, features = dfs(entityset=datetime_es,
```

Comment:
Maybe we could do 90 Days == 3 Months == 1 Quarter End?

Reply:
For this timestamp (2012-4-1), a QuarterEnd offset sets it to 1-31-2012. I can change the pd timedelta test so that it's consistent with the other two tests.